static/js/stream_list.js

```diff
@@ -435,19 +563,14 @@ $(function () {
             rebuild_recent_subjects(op_stream[0], subject);
             unread.process_visible();
         }
-        var op_pm = event.filter.operands('pm-with');
-        if (op_pm.length === 1) {
-            $("#user_presences li[data-email='" + op_pm[0] + "']").addClass('active-filter');
-        }
+
     });
 
     $(document).on('narrow_deactivated.zulip', function (event) {
-        if (zoomed_to_topics) {
-            zoom_out();
-        }
-        $("ul.filters li").removeClass('active-filter active-subject-filter');
-        remove_expanded_subjects();
+
+        reset_to_unnarrowed();
         $("#global_filters li[data-name='home']").addClass('active-filter');
+
```

Comment:
probably don't need this newline or the one that's now the first line of this block.